packages/protocol/contracts/stability/Exchange.sol

```diff
@@ -342,6 +342,7 @@ contract Exchange is IExchange, Initializable, Ownable, UsingRegistry, Reentranc
       registry.getAddressForOrDie(SORTED_ORACLES_REGISTRY_ID)
     )
       .medianRate(stable);
+    require(rateDenominator > 0, "exchange rate denominator cannot be less than or equal to 0");
```

Comment:
`exchange rate denominator cannot be 0` - no such thing as negative unsigned integers

Reply:
good point.  any reason to prefer `!==` over `>`?

Reply:
updated the error message